.github/workflows/main-push.yml

```
@@ -173,7 +173,53 @@ jobs:
+      packages: write
+    outputs:
+      image-tag: ${{ steps.tagger.outputs.image_tag }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)